includes/classes/Customer.php

```diff
@@ -380,7 +380,9 @@ public function banCustomer()
     public function setCustomerAuthorizationStatus(int $status)
     {
         global $db;
-        $db->Execute("UPDATE " . TABLE_CUSTOMERS . " SET customers_authorization = 4 WHERE customers_id=" . $this->customer_id, 1);
```

Comment:
I guess line 368 could be changed to call this function instead? (and remove the `global $db;` statement)

Reply:
we can